README.md

```diff
@@ -94,9 +94,15 @@ Moreover, having these extensions will help you to speed up the development proc
 
 ## How to contribute
 
-We have two types of PR (Pull request):
+To contribute in the project, we need to create PRs to master. We have two types of PRs (Pull request):
 
-### Base **Master**
+### PR Formating guideline
+
+1. Use the `[{Project Code}] {Developer}/{Clickup Card ID}/{Description}` format for PR titles. (e.g.: `[COJ] [Dev's Name]/COJ-247/Align next-button on mt5 modal`), where [COJ] is a clickup project code.
```

Comment:
I would add that the description should start with the verb in an imperative declaration (like in the example).

Reply:
Something like this?

1. Use the `[{Project Code}] {Developer}/{Clickup Card ID}/{Description}` format for PR titles (e.g., `[COJ] [Dev's Name]/COJ-247/Align next-button on mt5 modal`), where [COJ] is a ClickUp project code.

2. Start the description with a verb in an imperative declaration for clarity and conciseness. For example, "Fix issue with..." or "Implement feature to...".

Reply:
Would propose to be less verbal for fixes, eg `Fix login button disappears on resize`, `Fix user-is-not-defined on page load`

And much less verbal for features, eg `Render TrustPilot rating on build time`, `Move hotjar to partytown`, `Combine fonts into a single file`,  `Introduce new DIEL account confirmation workflow`